packages/cdk8s-cli/templates/python-app/.hooks.sscaff.js

```diff
@@ -47,8 +45,8 @@ exports.post = options => {
 
   chmodSync('main.py', '700');
 
-  execSync(`${cli} import k8s -l python`);
-  execSync(`pipenv run ./main.py`);
+  execSync(`cdk8s import k8s -l python`);
+  execSync(`pipenv run "./main.py"`);
```

Comment:
Getting nearly there. Failing here with:

```
'.' is not recognized as an internal or external command,
```

Likely some windows path magic to fix